.github/workflows/test_building_multiple_executables.yml

```diff
@@ -0,0 +1,32 @@
+name: test building multiple executables with CMake
+
+on:
+  pull_request:
+    paths:
+      - 'CMakeLists.txt'
```

Comment:
Will this run on only the base CMakeLists.txt changing or do we need `**/CMakeLists.txt` to run on any CMakeLists.txt changes no matter the base folder?

Reply:
My initial idea was to only check places that are relevant to build/post-build. But now I start to think it doesn't hurt to run this check on every PR, as a general CMake check. Will remove the rules here.